config.py

```diff
@@ -0,0 +1,10 @@
+user = "hritik"
+password = "****"
+host = "localhost"
+database = "mapbot"
+stanford_path_to_jar="./stanford-corenlp-3.9.2.jar"
+stanford_paht_to_models_jar="./stanford-corenlp-3.9.22-models.jar"
+#key = "*Google_Cloud_API_key*" #Will be provided by mentors
+#stanford_path_to_jar = "*your_path_to_stanford-corenlp-x.x.x.jar*" #for eg. "stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2.jar"
+#stanford_path_to_models_jar = "*your_path_to_stanford-corenlp-x.x.x-models.jar*" #for eg. "stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2-models.jar"
+#javahome = '*your_path_to_jdk_bin_java.exe*' #for eg. "C:/Program\ Files/Java/jdk1.8.0_201/bin/java.exe"
```

Comment:
Please remove this file, it needs to be added to .gitignore and never pushed as mentioned in the setup instructions [here](https://stackoverflow.com/questions/22682870/git-undo-pushed-commits)